external/CMakeLists.txt

```diff
@@ -61,10 +61,10 @@ set(OLP_SDK_CPP_ZLIB_URL "https://www.zlib.net/zlib-1.3.1.tar.gz")
 set(OLP_SDK_CPP_ZLIB_HASH "SHA256=9a93b2b7dfdac77ceba5a558a580e74667dd6fede4585b91eefb60f03b72df23")
 
 set(OLP_SDK_CPP_OPENSSL_URL "https://github.com/openssl/openssl.git")
-set(OLP_SDK_CPP_OPENSSL_TAG "OpenSSL_1_1_1w")
+set(OLP_SDK_CPP_OPENSSL_TAG "openssl-3.1.4")
```

Comment:
Openssl v3 has much more generated files than there is in cmake wrapper for openssl v1 so it will fail the build once forced. So we should either update that cmake wrapper for v3 and say goodbye for v1 or use openssl build system

Reply:
thanks for checking , tbd in next MRs